### ISSUE TITLE
:buffers should not show all tab groups by default

### ISSUE THREAD

**Issue by nathyong** [nathyong/dactyl#2]
If there is more than one tab group, then :buffers shows every buffer by default, sorted by groups. This is kind of jarring, vimperator only shows the active tab group. Perhaps there should be a switch to change this behaviour.

**Comment by Enzime** [comment]
I agree, but the issue with this change is workflow breaking. For the current tab group (with our defaults), we'd use e for the same functionality as b and E would be the same as B.
